posts/d3-tutorial-points-axes.md

```diff
@@ -0,0 +1,139 @@
+---
+authors:
+- Justin Risch
+tags:
+- d3.js
+- tutorial
+- data governance
+- data visualization
+date: 2017-08-17T18:22:50.000Z
+title: "Plotting Points on Two Axes with D3.JS"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2019/01/alignmentgraph.png
+---
+
+Welcome to this simple tutorial on how to plot a point on Two Axes, along with some nice formatting techniques. As well, we'll be using both linear scaling and string interpolation.
+
+## Setup
+
+1. Grab the source code from the following [link](https://github.com/JustinRisch/D3Example).
+2. After either downloading it as a zip or cloning it, use the following command in that directory to download the needed node\_modules packages: `npm install`
+   1. If you don't have npm installed, you can install it using [these instructions](https://www.npmjs.com/get-npm).
+   2. If it complains about permissions, try this command instead: `sudo npm install`
+3. Run the following command to start your Http Server: `http-server -p 8888`
+   1. If you don't have http-server installed, retry the above after running the command: `npm install http-server -g`
+
+You should now be able to navigate to the example UI by navigating to http://localhost:8888/src/alignment-axis.html
+
+## Features
+
+The code takes a linear scale from 0-4, maps those values to categories from the popular tabletop RPG "Dungeons & Dragons", then allows you to quickly plot a point on those axes by entering an X and Y value bounded by \[0,4\]. Additionally, the points plotted will be color scaled from Light Blue (Lawful / Good) to Dark Red (Chaotic / Evil). This is to make it a little prettier than using a standard black circle, as well as provide more visual feedback. Remember: visualization is about communicating data-- the more communication you have, the better!
+
+## The Code
```

Comment:
I think "The Code" should be a higher level header than "Variables and scales." through "The Circle(s)"

Reply:
Huh, it was meant to be. Thanks for catching that!